src/SignalR/clients/ts/signalr/src/index.ts

```diff
@@ -3,7 +3,7 @@
 
 // Everything that users need to access must be exported here. Including interfaces.
 export { AbortSignal } from "./AbortController";
-export { AbortError, HttpError, TimeoutError } from "./Errors";
+export { AbortError, HttpError, TimeoutError, AggregateErrors, DisabledTransportError, FailedToStartTransportError, UnsupportedTransportError } from "./Errors";
```

Comment:
These shouldn't be exported unless you want to bring them to API review

Reply:
If it's just for this one use case, may not make much sense to expose all these additional types (especially considering they aren't comprehensive on all potential HttpConnection exceptions). Updated https://github.com/dotnet/aspnetcore/pull/34766/commits/666db5a18ad7dccc34163179f2d0476266aedf77.